packages/js/wasm/src/helpers/createWasmWrapper.ts

```diff
@@ -0,0 +1,62 @@
+import { IFileReader, WasmWrapper } from "..";
+import { createWasmPackage } from "./createWasmPackage";
+
+import { GetManifestOptions } from "@polywrap/core-js";
+
+// Helper function for creating wasm wrappers
+// Not meant for exporting out of this library
+export const createWasmWrapper = async (
+  manifestBufferOrFileReader: Uint8Array | IFileReader,
+  wasmModuleOrFileReaderOrManifestOptions?:
+    | Uint8Array
+    | IFileReader
+    | GetManifestOptions,
+  fileReaderOrManifestOptions?: IFileReader | GetManifestOptions,
+  manifestOptions?: GetManifestOptions
+): Promise<WasmWrapper> => {
+  if (
+    !wasmModuleOrFileReaderOrManifestOptions ||
+    (wasmModuleOrFileReaderOrManifestOptions as GetManifestOptions)
```

Comment:
Can this test be (wasmModuleOrFileReaderOrManifestOptions as GetManifestOptions).noValidate !== undefined? I'm guessing you already thought of that, but I'm just checking. 😄

Reply:
This makes sense to me!

Reply:
Yep, this was a blunder, nice catch!